From 82a697fdd09beb673d94ed197ea1dd527b070cae Mon Sep 17 00:00:00 2001
From: Matthew Fatheree <Matthew.Fatheree@belkin.com>
Date: Sat, 3 May 2014 22:59:46 +0700
Subject: [PATCH 01/34] Add Linksys Mamba WRT1900AC build profile

Add Linksys Mamba WRT1900AC build profile to target/linux/mvebu
Signed-off-by: Matthew Fatheree <Matthew.Fatheree@belkin.com>
---
 target/linux/mvebu/profiles/101-Linksys-Mamba.mk |   21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)
 create mode 100644 target/linux/mvebu/profiles/101-Linksys-Mamba.mk

diff --git a/target/linux/mvebu/profiles/101-Linksys-Mamba.mk b/target/linux/mvebu/profiles/101-Linksys-Mamba.mk
new file mode 100644
index 0000000..de048a6
--- /dev/null
+++ b/target/linux/mvebu/profiles/101-Linksys-Mamba.mk
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2013 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+define Profile/Mamba
+  NAME:=Linksys Mamba
+  PACKAGES:= \
+	kmod-usb-storage \
+	kmod-of-i2c kmod-i2c-core kmod-i2c-mv64xxx \
+	kmod-ata-core kmod-ata-marvell-sata \
+	kmod-rtc-marvell kmod-thermal-armada
+endef
+
+define Profile/Mamba/Description
+ Package set compatible with Linksys Mamba board (WRT1900AC board.)
+endef
+
+$(eval $(call Profile,Mamba))
-- 
1.7.9.5

